homeassistant/components/weheat/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "weheat",
+  "name": "Weheat",
+  "codeowners": ["@jesperraemaekers"],
+  "config_flow": true,
+  "dependencies": ["application_credentials"],
+  "documentation": "https://www.home-assistant.io/integrations/weheat",
+  "homekit": {},
+  "iot_class": "cloud_polling",
+  "requirements": ["weheat==2024.7.8rc1"],
```

Comment:
The latest release is `2024.7.8`, no need for rc1

Reply:
Yea that was also something I noticed

Reply:
This has been resolved as the version was bumped.